backend/progetto-risiko-risikiamo-backend/src/main/java/com/project/progettorisikorisikiamobackend/obiettivi/Obiettivi.java

```diff
@@ -0,0 +1,59 @@
+package com.project.progettorisikorisikiamobackend.obiettivi;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.security.SecureRandom;
+
+
+public class Obiettivi {
+    HashMap<Integer, Enum> objectives=null;
```

Comment:
E' codice un pò troppo rigido, prova uno strategy per un approccio più modulare

Reply:
Ho implementato uno strategy che diversifica i tipi di obiettivi in modo che possano essere più flessibili

Reply:
La DOC sarebbe carina